autoinstrumentation/python/requirements.txt

```diff
@@ -1,4 +1,4 @@
-opentelemetry-distro==0.51b0
```

Comment:
all the versons below should be updated as well

Reply:
I updated them manually.